src/lua/msgpack.c

```diff
@@ -519,6 +521,16 @@ luamp_decode_with_ctx(struct lua_State *L, struct luaL_serializer *cfg,
 			VERIFY(interval_unpack(data, len, itv) != NULL);
 			return;
 		}
+		case MP_ARROW:
+		{
+			struct arrow_record_batch *arr =
+				luaT_new_arrow_record_batch(L);
+			if (arrow_record_batch_unpack(data, len, arr) == NULL) {
+				/* Diag is set by arrow_record_batch_unpack() */
+				luaT_error(L);
+			}
+			return;
```

Comment:
Let's also introduce encode and check decode/encode returns same hex.

Reply:
Currently the encoding to msgpack is useless and requires some refactoring, will do it later.